editor/src/EditorStartup.cpp

```diff
@@ -245,6 +243,7 @@ void EditorStartup::pauseGame()
             auto updateSceneManagerFunct = GET_PROCESS(m_reloadableCpp, updateSceneManager);
             updateSceneManagerFunct();
             m_engine->inputManager.processInput();
+            m_engine->soundSystem.update();
```

Comment:
There never is a situation when we can play sound in the editor while the game simulation is paused.
I believe it is unnecessary to update the sound system here.

```suggestion
```

Reply:
same than before